packages/cashier/src/pages/account-transfer/__tests__/account-transfer.spec.tsx

```diff
@@ -271,30 +270,4 @@ describe('<AccountTransfer />', () => {
 
         expect(await screen.findByText(/You have no funds/i)).toBeInTheDocument();
     });
-
-    it('should show the receipt if transfer is successful', async () => {
-        const mock_root_store = mockStore({
-            client: {
-                mt5_login_list: [
-                    {
-                        account_type: 'demo',
-                        sub_account_type: 'financial_stp',
-                    },
-                ],
-            },
-            modules: {
-                cashier: {
-                    ...cashier_mock,
-                    account_transfer: {
-                        ...cashier_mock.account_transfer,
-                        is_transfer_confirm: true,
-                    },
-                },
-            },
-        });
-
-        renderAccountTransfer(mock_root_store);
-
-        expect(await screen.findByText('mockedAccountTransferReceipt')).toBeInTheDocument();
```

Comment:
`<AccountTransferReceipt />` is not part of `<AccountTransfer />` anymore, hence removing the invalid test.